README.md

```diff
@@ -1,3 +1,114 @@
+
+## Ignition Setup ##
+*Information for setup of Ignition (steps 1 - 4) can also be found here: [https://docs.inductiveautomation.com/display/DOC80/Installing+and+Upgrading](https://docs.inductiveautomation.com/display/DOC80/Installing+and+Upgrading "https://docs.inductiveautomation.com/display/DOC80/Installing+and+Upgrading")*
+
+**Step 1:** Using a web browser, download **Ignition 8** from [https://inductiveautomation.com/downloads/ignition](https://inductiveautomation.com/downloads/ignition "https://inductiveautomation.com/downloads/ignition").
+
+**Step 2:** After the download is complete, run the Ignition installer. The default installation options are satisfactory, although you may change them to fit your preference(s).
+
+**Step 3:** After Ignition is installed, the Ignition Gateway web page will open and guide you through commissioning Ignition. If the Ignition Gateway web page does not automatically open, using your browser, navigate to [http://localhost:8088](http://localhost:8088 "http://localhost:8088"). Take note of the username and password you set during commissioning, as this will be required in later setup steps.
+
+**Step 4:** After Ignition is commissioned, you will be redirected to the Ignition Gateway homepage. Take note of the URL in your browser's address bar as this will be required in later setup steps.
+
+**Step 5:** Download the HMS Ewon Connector from the EwonConnector releases page, found here: [https://github.com/hms-networks/EwonConnector/releases](https://github.com/hms-networks/EwonConnector/releases "https://github.com/hms-networks/EwonConnector/releases"). To download, expand the `Assets` menu under the latest Ignition 8 release, and click on the first file, commonly named `EwonConnection-X.Y.Z.zip`, where X.Y.Z is the version number.
+
+**Step 6:** After the download is complete, install the HMS Ewon Connector by going to the `Config` tab in Ignition, clicking `Modules` and then choosing `Install or Upgrade a Module...`. Click `Choose File` and select the file you've just downloaded, then click `Install`.
+
+**Step 7:** After installation of the HMS Ewon Connector, browse to the Ewon Connector configuration by going to the `Config` tab in Ignition, and clicking `Ewon Connector`, located under the `Tags` heading. *NOTE: the default name "Ewon" should NOT be changed*.
+
+**Step 8:** Ensure that the `Enabled` check-box is checked.  
+
+**Step 9:** Set the `Live Data Poll Rate in Seconds` value to 1 second.
+
+**Step 10:** Ensure that the `Read all values in realtime` check-box is checked.
+
+**Step 11:** Under `Ewon Account Information`, input your Talk2M account information. All fields are required. Please note that you may need to select the `Change Password?` option to enable the password fields. If you do not have a Talk2M/Ewon API Key, you may request one here: [https://developer.ewon.biz/registration](https://developer.ewon.biz/registration "https://developer.ewon.biz/registration").
```

Comment:
This almost should be in the Ewon Setup area as it takes a bit of time to get the DevID.  If they complete this step early they may have their DevID by the time they get here.

Reply:
Moved to the list of "required items" under Getting Started